rachellegelden/Home-Together-Canada
# HTC-159: Login css

# [Issue 159](https://github.com/rachellegelden/Home-Together-Canada/issues/159)

closes #159 

## Summary
- added new images for login page 
- added css to login, created loginForm.css
-  updated submitButton in BusinessRegistrationForm due to new prop 
- moved some CSS from Home.css to components.css to share in the whole project
- created new GenericInput component, with test and snapshot
- updated submitButton, TextArea with className, updated their tests and snapshots
- moved login to its own folder in src

## Relevant Motivation & Context
- Make login page pretty

## Testing Instructions
- Run `npm test` to ensure all tests are passing
- Run `npm start` to review login css

## Developer checklist prior to opening this pull request:


## Reviewer
